src/navigations/root-stack.js

```diff
@@ -97,29 +97,11 @@ export const RootNavigator = ({currentScreen}) => {
 
   const isUnauthenticated = initialStartup.id === null || initialStartup.id === '';
 
-  const getPaddingTop = (screenName) => {
-    'worklet';
-
-    if (isUnauthenticated || ['TopicPageScreen', 'TopicMemberScreen'].includes(screenName)) {
-      return 0;
-    }
-    return insets.top;
-  };
-
-  const getInsetTopColor = () => {
-    'worklet';
-
-    return currentScreen === 'SampleChatScreen' ? colors.anon_primary : colors.white;
-  };
-
   return (
     <LoadingStartupContext.Provider value={loadingStartup.loadingUser}>
       <View
         style={{
-          height: '100%',
-          paddingBottom: isUnauthenticated ? 0 : insets.bottom,
-          paddingTop: getPaddingTop(currentScreen),
-          backgroundColor: getInsetTopColor()
+          height: '100%'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from dynamic padding and background color to static values may cause visual inconsistencies across different screens. If the previous logic was necessary for proper display on certain screens, this change could introduce visual bugs.
```diff
-  const getPaddingTop = (screenName) => {
-    'worklet';
-
-    if (isUnauthenticated || ['TopicPageScreen', 'TopicMemberScreen'].includes(screenName)) {
-      return 0;
-    }
-    return insets.top;
-  };
-
-  const getInsetTopColor = () => {
-    'worklet';
-
-    return currentScreen === 'SampleChatScreen' ? colors.anon_primary : colors.white;
-  };
-
-   return (
-     <LoadingStartupContext.Provider value={loadingStartup.loadingUser}>
-       <View
-         style={{
-          height: '100%',
-          paddingBottom: isUnauthenticated ? 0 : insets.bottom,
-          paddingTop: getPaddingTop(currentScreen),
-          backgroundColor: getInsetTopColor()
-         }}>
-         <NetworkStatusIndicator hide={true} />
+   return (
+     <LoadingStartupContext.Provider value={loadingStartup.loadingUser}>
+       <View
+         style={{
+           height: '100%'
+         }}>
+         <NetworkStatusIndicator hide={true} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->